.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 
 python:
   - "2.7"
+  - "3.5"
```

Comment:
if we'd like be sure that it works out of the box on current debian stable and up, then 3.4 should be added too.